pkg/interceptors/cel/cel_test.go

```diff
@@ -203,7 +204,7 @@ func TestInterceptor_ExecuteTrigger(t *testing.T) {
 				rt.Fatalf("error reading response body: %v", err)
 			}
 			defer resp.Body.Close()
-			if !reflect.DeepEqual(got, tt.want) {
+			if diff := cmp.Diff(got, tt.want); diff != "" {
```

Comment:
Small nit: Usually, we'd want to  do something like the following:
```
if diff := cmp.Diff(got, tt.want); diff != "" {
  t.Errorf("FuncBeingTestedName() returned diff (-want +got): %s
}
```

This way we are consistent - the `want` always comes before `got` and the `diff` prints out with missing lines starting with a - and extra lines with a +

Reply:
Modified
Thank you